backend/src/main/java/se/chalmers/dat265/group1/api/order/OrderController.java

```diff
@@ -73,6 +88,22 @@ public Order updateOrder(String orderID, Order order) {
         return getOrder(orderID);
     }
 
+    /**
+     * Dates should be on format yyyy-MM-dd
+     *
+     * @param date
+     */
+    private void validateDate(String date) throws InvalidDateFormatException {
+        //TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)